app/views/katello/api/v2/products/base.json.rabl

```diff
@@ -12,6 +12,8 @@ attributes :sync_summary
 attributes :gpg_key_id
 attributes :redhat? => :redhat
 
+attributes :available_content => :available_content, :if => params[:include_available_products]
```

Comment:
Where is this parameter defined? If its new, the API docs for the action would need to include it